be/src/storage/tablet_updates.cpp

```diff
@@ -861,16 +861,36 @@ void TabletUpdates::_ignore_rowset_commit(int64_t version, const RowsetSharedPtr
                               << " txn_id: " << rowset->txn_id() << " rowset: " << rowset->rowset_id().to_string();
 }
 
+// check error message to avoid error code conversion during execution
+bool TabletUpdates::_check_status_msg(std::string_view msg) {
```

Comment:
Add UT for this?

Reply:
> Add UT for this?

not finished yet, I will add some test later.